diagnostics/inspection.py

```diff
@@ -24,46 +24,48 @@
 from . import utils
 
 
+sys.setrecursionlimit(5000)
```

Comment:
And to head off the obvious question: I was getting some recursion errors in the "true" (dt=2^-14) simulation. That's a ludicrously small step size so I don't think it's an issue generally speaking.

Reply:
Sounds good. A `dt` that small is almost never used in practice, though as a diagnostic for the rate, it should still be worthwhile to go this far.